docs/index.md

```diff
@@ -551,7 +551,7 @@ Keccak-256 hash of the RLP of null  (a `String`)
 
 Type: [String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)
 
-## lsetLength
+## setLengthLeft
```

Comment:
This documentation is auto generated. If this isn't part of the source file, how can this end up here?